apps/webhook/src/webhooks/usecases/webhook/webhook.command.ts

```diff
@@ -0,0 +1,13 @@
+import { IsDefined } from 'class-validator';
+import { EnvironmentCommand } from '@novu/application-generic';
+
+export class WebhookCommand extends EnvironmentCommand {
+  @IsDefined()
+  providerId: string;
+
+  @IsDefined()
+  body: any;
+
+  @IsDefined()
+  type: 'sms' | 'email';
```

Comment:
Are these the only notification types that are going to use webhooks?

Reply:
Right now I think we will go with those two and then there can be more in the future 😄